content/en/docs/contribute/style/style-guide.md

```diff
@@ -210,6 +210,11 @@ The output is similar to this:
     NAME     READY     STATUS    RESTARTS   AGE    IP           NODE
     nginx    1/1       Running   0          13s    10.200.0.4   worker0
 
+
+### Use shortcode for downloadable code blocks
+
+Use [codenew](#downloadable-code-blocks) shortcode for downloadable code blocks.
```

Comment:
```suggestion
Use a [codenew](#downloadable-code-blocks) shortcode for code blocks that are ready to run.
```
(if a code block is only an excerpt, it doesn't need to be downloadable)

It might go without saying: it's also a good idea to advise authors to tell the reader about how to run that.
I've seen pages that suggest `kubectl apply -f <url>`; should the style guide mention that bit too?

Reply:
updated. Though not very sure if the language I have used for suggesting this is great. Looking forward for this review.